mubench.reviewsite/tests/CSVHelperTest.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+use Monolog\Logger;
+use PHPUnit\Framework\TestCase;
+use MuBench\ReviewSite\Model\ExperimentResult;
+use MuBench\ReviewSite\Model\DetectorResult;
+use MuBench\ReviewSite\Model\Detector;
+use MuBench\ReviewSite\Model\Misuse;
+use \MuBench\ReviewSite\CSVHelper;
+
+class CSVHelperTest extends TestCase
+{
+
+    private $csv_helper;
+
+    function setUp()
+    {
+        $this->csv_helper = new CSVHelper();
+    }
+
+    public function test_headers_for_stats_ex1()
```

Comment:
One of the headers is tested again below. For the other experiments, you don't test that the values are correct. Please write one test per experiment that checks the entire output. Preferably with some real values (different from 0, I mean).